refactor: updated CI badges

rutvik-panchal opened the pull request:
Fixes #1576 

Please Add Screenshots If there are any UI changes.
NA

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

rutvik-panchal committed:
refactor: updated CI badges

rutvik-panchal commented:
@miPlodder Updated the link for CI badges.

miPlodder reviewed (commented):
Can you take care of the 2nd item (about adding instructions to generate APK in the separate heading (level -3 or whatever looks appropriate)), I mentioned in the issue.

rutvik-panchal commented:
> Can you take care of the 2nd item (about adding instructions to generate APK in the separate heading (level -3 or whatever looks appropriate)), I mentioned in the issue.

I have added the heading and updated the PR.
